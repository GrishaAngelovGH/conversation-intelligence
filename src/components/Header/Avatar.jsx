const Avatar = () => (
  <img src='https://randomuser.me/api/portraits/men/18.jpg' width={50} height={50} className='rounded-circle ms-1' alt='user' />
)

export default Avatar