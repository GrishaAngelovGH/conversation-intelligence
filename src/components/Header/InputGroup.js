import Form from 'react-bootstrap/Form'
import Button from 'react-bootstrap/Button'
import InputGroup from 'react-bootstrap/InputGroup'

const InputGroupWrapper = () => (
  <InputGroup>
    <Form.Control placeholder="Search for keywords, brands, tags and more" />
    <InputGroup.Text><i className="bi bi-search"></i></InputGroup.Text>
  </InputGroup>
)

export default InputGroupWrapper