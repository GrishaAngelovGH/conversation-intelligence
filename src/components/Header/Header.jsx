import Avatar from './Avatar'
import InputGroup from './InputGroup'

const Header = () => {
  const style = { fontSize: 30 }

  return (
    <div className='row g-0 bg-primary p-2 justify-content-between align-items-center'>
      <div className='col-1 col-md-1'>
        <i className='bi bi-grid-3x3-gap text-white' style={style}></i>
      </div>

      <div className='col-5 col-md-4'>
        <InputGroup />
      </div>

      <div className='col-3 col-md-1 d-flex'>
        <i className='bi bi-gear text-white' style={style}></i>
        <Avatar />
      </div>
    </div>
  )
}

export default Header