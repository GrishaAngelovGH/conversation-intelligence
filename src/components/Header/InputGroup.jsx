import Form from 'react-bootstrap/Form'
import InputGroup from 'react-bootstrap/InputGroup'

const InputGroupWrapper = () => (
  <InputGroup>
    <Form.Control placeholder='Search for keywords, brands, tags and more' />
    <InputGroup.Text><i className='bi bi-search'></i></InputGroup.Text>
  </InputGroup>
)

export default InputGroupWrapper