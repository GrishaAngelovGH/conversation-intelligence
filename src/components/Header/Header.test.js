import { render } from '@testing-library/react'

import Header from './Header'

test('should render component', () => {
  const container = render(<Header />)

  expect(container).toMatchSnapshot()
})