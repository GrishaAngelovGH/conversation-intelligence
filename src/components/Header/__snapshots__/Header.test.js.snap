// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="row g-0 bg-primary p-2 justify-content-between align-items-center"
      >
        <div
          class="col-1 col-md-1"
        >
          <i
            class="bi bi-grid-3x3-gap text-white"
            style="font-size: 30px;"
          />
        </div>
        <div
          class="col-5 col-md-4"
        >
          <div
            class="input-group"
          >
            <input
              class="form-control"
              placeholder="Search for keywords, brands, tags and more"
            />
            <span
              class="input-group-text"
            >
              <i
                class="bi bi-search"
              />
            </span>
          </div>
        </div>
        <div
          class="col-3 col-md-1 d-flex"
        >
          <i
            class="bi bi-gear text-white"
            style="font-size: 30px;"
          />
          <img
            alt="user"
            class="rounded-circle ms-1"
            height="50"
            src="https://randomuser.me/api/portraits/men/18.jpg"
            width="50"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="row g-0 bg-primary p-2 justify-content-between align-items-center"
    >
      <div
        class="col-1 col-md-1"
      >
        <i
          class="bi bi-grid-3x3-gap text-white"
          style="font-size: 30px;"
        />
      </div>
      <div
        class="col-5 col-md-4"
      >
        <div
          class="input-group"
        >
          <input
            class="form-control"
            placeholder="Search for keywords, brands, tags and more"
          />
          <span
            class="input-group-text"
          >
            <i
              class="bi bi-search"
            />
          </span>
        </div>
      </div>
      <div
        class="col-3 col-md-1 d-flex"
      >
        <i
          class="bi bi-gear text-white"
          style="font-size: 30px;"
        />
        <img
          alt="user"
          class="rounded-circle ms-1"
          height="50"
          src="https://randomuser.me/api/portraits/men/18.jpg"
          width="50"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
